### DISCUSSION TITLE
Is it possible to hide the date from the tooltip for files, tasks, and email?

### DISCUSSION THREAD

**Discussion by absuptAlmbrand** [deejay-hub/timeline-lwc#154] [Q&A] [answered]
As the subject says. Is there any way to hide date from the tooltip on files, tasks, and emails?
![image](https://user-images.githubusercontent.com/58294515/139436328-2cc8c685-9e1f-494d-862b-7324299cba02.png)

**Comment by deejay-hub** [comment, marked as answer]
No. The date used to plot the record is always shown for any object not supported by the UI API in v1.9 and above. This includes files and activities (events and tasks).

We'll probably wait for UI API support for these objects so we can simply use the compact layout in the future but no ETA from Salesforce on when this might be.
